### PR TITLE
[zh-cn] Update Firefox experimental features

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Finish translation of firefox experimental features

### Motivation

Get rid of half translated files

### Related issues and pull requests

#14272

### CODE REVIEW

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -1,656 +1,2713 @@
...
+      <td colspan="2">
+      <code>layout.css.abs-sign.enabled</code>
+    </td>
+    </tr>
+  </tbody>
+</table>
+
+### rect() 和 xywh() 基本图形函数
+
+通过 CSS [`rect()`](/zh-CN/docs/Web/CSS/basic-shape/rect) 和 [`xywh()`](/zh-CN/docs/Web/CSS/basic-shape/xywh) 形状函数，你可以使用 [`<basic-shape>`](/zh-CN/docs/Web/CSS/basic-shape) 数据类型定义矩形。在 CSS 属性（如 {{cssxref("offset-path")}}） 中，这些函数用于定义元素移动路径的形状。使用 `rect()` 函数，可以指定矩形边缘与包含块的顶部边缘和左侧边缘的偏移量。使用 `xywh()` 函数，可以指定矩形边缘与包含块的左边缘和顶边缘的偏移量，以及矩形的宽度和高度。在这两个函数中，都可以选择对边角进行舍入。有关详细信息，请参阅 `rect()` 函数的 [Firefox bug 1786161](https://bugzil.la/1786161) 和 `xywh()` 函数的 [Firefox bug 1786160](https://bugzil.la/1786160)。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/firefox/experimental_features/index.md:961]:
```suggestion
通过 CSS [`rect()`](/zh-CN/docs/Web/CSS/basic-shape/rect) 和 [`xywh()`](/zh-CN/docs/Web/CSS/basic-shape/xywh) 形状函数，你可以使用 [`<basic-shape>`](/zh-CN/docs/Web/CSS/basic-shape) 数据类型定义矩形。在 CSS 属性（如 {{cssxref("offset-path")}}）中，这些函数用于定义元素移动路径的形状。使用 `rect()` 函数，可以指定矩形边缘与包含块的顶部边缘和左侧边缘的偏移量。使用 `xywh()` 函数，可以指定矩形边缘与包含块的左边缘和顶边缘的偏移量，以及矩形的宽度和高度。在这两个函数中，都可以选择对边角进行舍入。有关详细信息，请参阅 `rect()` 函数的 [Firefox bug 1786161](https://bugzil.la/1786161) 和 `xywh()` 函数的 [Firefox bug 1786160](https://bugzil.la/1786160)。
```

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -1,656 +1,2713 @@
...
+      <td>117</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>正式发布版</th>
+      <td>117</td>
+      <td>否</td>
+    </tr>
+    <tr>
+      <th>首选项名称s</th>
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/firefox/experimental_features/index.md:993]:
```suggestion
      <th>首选项名称</th>
```

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 3]
```diff
@@ -1,656 +1,2713 @@
...
+    <tr>
+      <th>首选项名称</th>
+      <td colspan="2">
+        <code>dom.security.featurePolicy.header.enabled</code>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Clear-Site-Data “cache”指令
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/firefox/experimental_features/index.md:2325]:
```suggestion
### Clear-Site-Data“cache”指令
```

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 4]
```diff
@@ -1,656 +1,2713 @@
...
+      <td>-</td>
+    </tr>
+    <tr>
+      <th>首选项名称</th>
+      <td colspan="2"><code>javascript.options.experimental.well_formed_unicode_strings</code></td>
+    </tr>
+  </tbody>
+</table>
+
+## APIs
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/firefox/experimental_features/index.md:1167]:
```suggestion
## API
```

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 5]
```diff
@@ -1,656 +1,2713 @@
...
+      <td colspan="2"><code>accessibility.ARIAReflection.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
 
 ## CSS
 
+### 显示杂散控制字符的十六进制框
+
+该特性会将除 _tab_（`U+0009`）、_line feed_（`U+000A`）、_form feed_（`U+000C`）和 _carriage return_（`U+000D`）之外的控制字符（Unicode 类别 Cc）渲染为十六进制框，而这些控制字符并非预期字符。（详见 [Firefox bug 1099557](https://bugzil.la/1099557)）。
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/experimental_features/index.md:142]: could we translate the key name:

```suggestion
该特性会将除*制表符*（`U+0009`）、_换行符_（`U+000A`）、_换页符_（`U+000C`）和*回车符*（`U+000D`）以外的控制字符（Unicode 类别 Cc）渲染为十六进制框，而这些控制字符并非预期字符。（详见 [Firefox bug 1099557](https://bugzil.la/1099557)）。
```

I'm not sure about the translation of "hexbox".

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 6]
```diff
@@ -1,656 +1,2713 @@
...
-| **日期和时间输入**{{HTMLElement("input")}}为 [日期](/zh-CN/docs/Web/HTML/Element/input/date) 和 [时间](/zh-CN/docs/Web/HTML/Element/input/time) 相关的输入类型提供的新用户界面 | 开启 <sub>(从 51 开始支持，从 55 开始默认开启)</sub> | 开启 <sub>(从 51 开始支持)</sub> | 关闭 <sub>(从 51 开始支持)</sub> | ---            | `dom.forms.datetime`         |
-| **{{HTMLElement("dialog")}}** 包含与之交互的 DOM 元素的对话框元素。Implementation of modal dialogs and accessibility still missing.                                            | 关闭 <sub>(从 53 开始支持)</sub>                     | 关闭 <sub>(从 53 开始支持)</sub> | ---                              | ---            | `dom.dialog_element.enabled` |
+### input type="search" 外观
+
+输入类型 `type="search"` 的布局已更新。一旦有人开始输入，搜索字段就会有一个清晰的图标，以便与其他浏览器的实现相匹配。（详情请参见 [Firefox bug 558594](https://bugzil.la/558594)）。
+
+<table>
+  <thead>
+    <tr>
+      <th>发布渠道</th>
```
[the remark this answers is not in the document]
Comment (zhoushengdao) [reply files/zh-cn/mozilla/firefox/experimental_features/index.md:25]: 我看到 Firefox 上写的是“更新通道”。

image

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 7]
```diff
@@ -1,656 +1,2713 @@
...
 slug: Mozilla/Firefox/Experimental_features
 ---
 
 {{FirefoxSidebar}}
 
-Mozilla 发布了测试的 nightly 版本，用于测试新的浏览器特性。实验性特性，例如所提议的可用的 Web 平台标准的实现。这个页面列出了 Firefox Nightly 中的一些新特性和启用方法等信息。在发布这些功能之前，你可以测试 Web 站点和应用程序，并确保一切都能与最新的 Web 技术功能协同工作。
+本页列出了 Firefox 的实验性和部分实现的功能，包括针对提议的或前沿的网络平台标准的功能，以及这些功能的版本信息、是否“默认”激活这些功能，以及可用于激活或停用这些功能的偏好设置。这样，就可以在功能发布前对其进行测试。
 
-你需要下载隔夜版或是开发者版，来测试下面这些实验性特性。
+新功能首先出现在 [Firefox Nightly](https://www.mozilla.org/zh-CN/firefox/channel/desktop/) 版本中，通常默认启用。它们随后会传播到 [Firefox 浏览器开发者版](https://www.mozilla.org/zh-CN/firefox/developer/)，并最终传播到发布版本中。在发布版中默认启用某项特性后，该功能就不再被视为实验性特性，应从主题中删除。
```
[the remark this answers is not in the document]
Comment (jasonren0403) [reply files/zh-cn/mozilla/firefox/experimental_features/index.md:10]:
```suggestion
新特性首先出现在 [Firefox Nightly](https://www.mozilla.org/zh-CN/firefox/channel/desktop/) 版本中，通常默认启用。它们随后会引入到 [Firefox 浏览器开发者版](https://www.mozilla.org/zh-CN/firefox/developer/)，并最终引入到发布版本中。在发布版中默认启用某项特性后，该功能就不再被视为实验性特性，应从主题中删除。
```

File: files/zh-cn/mozilla/firefox/experimental_features/index.md [hunk 8]
```diff
@@ -1,656 +1,2713 @@
...
+    <tr>
+      <th>首选项名称</th>
+      <td colspan="2"><code>layout.forms.input-type-show-password-button.enabled</code></td>
+    </tr>
+  </tbody>
+</table>
+
+### ARIA 属性反射
+
+为非 IDREF 属性启用了 [ARIA](/zh-CN/docs/Web/Accessibility/ARIA) 反射功能，允许作者通过 JavaScript API 直接获取和设置 DOM 元素上的 ARIA 属性，而不是使用 `setAttribute` 和 `getAttribute`（详情请参见 [Firefox bug 1824980](https://bugzil.la/1824980)）。
```
Comment (jasonren0403) [inline files/zh-cn/mozilla/firefox/experimental_features/index.md:100]:
```suggestion
为非 IDREF 属性启用了 [ARIA](/zh-CN/docs/Web/Accessibility/ARIA) 反射特性，允许作者通过 JavaScript API 直接获取和设置 DOM 元素上的 ARIA 属性，而不是使用 `setAttribute` 和 `getAttribute`（详情请参见 [Firefox bug 1824980](https://bugzil.la/1824980)）。
```

File: files/zh-cn/mozilla/firefox/experimental_features/index.md (hunk 5 above)
Comment (jasonren0403) [reply files/zh-cn/mozilla/firefox/experimental_features/index.md:142]: Some old bug report document have some screenshots, the malfunctioned UI text look like 'hexbox'es 👀 

image

Comment (yin1999) [reply files/zh-cn/mozilla/firefox/experimental_features/index.md:142]: Thanks for the clarification @jasonren0403 :+1:

**Review by yin1999 — APPROVED** [review]
Thanks @jasonren0403

**Pull request merged**